samples/apps/copilot-chat-app/webapi/CopilotChat/Controllers/ChatHistoryController.cs

```diff
@@ -25,26 +26,30 @@ namespace SemanticKernel.Service.CopilotChat.Controllers;
 public class ChatHistoryController : ControllerBase
 {
     private readonly ILogger<ChatHistoryController> _logger;
-    private readonly ChatSessionRepository _chatSessionRepository;
-    private readonly ChatMessageRepository _chatMessageRepository;
+    private readonly ChatSessionRepository _sessionRepository;
+    private readonly ChatMessageRepository _messageRepository;
     private readonly PromptsOptions _promptOptions;
+    private readonly ChatMemorySourceRepository _sourceRepository;
 
     /// <summary>
     /// Initializes a new instance of the <see cref="ChatHistoryController"/> class.
     /// </summary>
     /// <param name="logger">The logger.</param>
     /// <param name="chatSessionRepository">The chat session repository.</param>
     /// <param name="chatMessageRepository">The chat message repository.</param>
+    /// <param name="chatMemorySourceRepository">The chat memory resource repository.</param>
```

Comment:
The parameter names in the XMLDoc do not match the actual parameter names. I am surprised that the IDE didn't catch this.

Reply:
good catch, fixed